src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -0,0 +1,59 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import unittest
+
+import sas
+from util import winnforum_testcase
+
+
+class SpectrumInquiryTestcase(unittest.TestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_10(self):
```

Comment:
On Pg 16 of the spec (version 5.5.1) this portion of the value can be either WINNF or FCC. I am guessing that the WINFF found later in that document is a typo, and will be updated at some time. I suggest we should go with what is correct, rather that following the name WINFF blindly. For now, I have changed it to WINFF, however, please let me know if we can agree on using WINNF.